README.md

```diff
@@ -196,7 +196,19 @@ You will also need to enable and configure the Grafana `dashboards sidecar` like
 
 The `k8s-views-nodes` dashboard will have many broken panels if the `node` label from `kube_node_info` doesn't match the `nodename` label from `node_uname_info`.
 
-This issue being discussed in [#41](https://github.com/dotdc/grafana-dashboards-kubernetes/pull/41).
+This situation can happen on certain deployments of the node exporter running inside Kubernetes, where `nodename` takes a different value than the node name as understood by the Kubernetes API.
+
+Assuming the node exporter target is defined through `kubernetes_sd_configs` (e.g. via a `ServiceMonitor`), you can take advantage of the internal discovery labels and fix this by writing the following relabeling rule:
+
+```yaml
+action: replace
+sourceLabels:
+- __meta_kubernetes_pod_node_name
+targetLabel: nodename
+```
+
```

Comment:
Add:

This syntax is for Prometheus Operator, for Prometheus:

`sourceLabels ` -> `source_labels`
`targetLabel ` -> `target_tabel`